### PR TITLE
Feature/scat 964 procurement overview

### PR DESCRIPTION
**Pull request by CesarPerez**
### JIRA link
https://crowncommercialservice.atlassian.net/browse/SCAT-964

### Change description

Added template to show the procurement, the first time you enter in the lot the project is created but is not fully ready yet (we manage to get the response from the post endpoint but we need to display the data on the procurement template tomorrow), 

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Check and send page updated
- [x] UI changes look good on mobile

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by ulysses-cognizant — APPROVED** [review]

File: src/main/common/middlewares/agreementservice/chooseagreement.ts [hunk 1]
```diff
@@ -14,13 +13,12 @@ import { sortObject } from '../../util/operators/sortObject';
  */
 export class ChooseAgreementMiddleware {
     static FetchAgreements = (req: express.Request, res: express.Response, next: express.NextFunction) => {
-        let BaseURL = "/agreements/RM1043.6";
+        let BaseURL = "/agreements/RM1043.6/lots";
```
Comment (prathap-cognizant) [inline src/main/common/middlewares/agreementservice/chooseagreement.ts:16]: We need to change it to RM6263

File: src/main/features/dashboard/controller/dashboard.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import * as dashboarData from '../../../resources/content/dashboard/ccs-dashboar
  * @param res 
  */
 export const DASHBOARD = (req : express.Request, res : express.Response)=> {
+  res.cookie('agreement_id', 'RM1043.6', { maxAge: 900000, httpOnly: true });
```
Comment (prathap-cognizant) [inline src/main/features/dashboard/controller/dashboard.ts:12]: need to change it to RM6263

**Review by prathap-cognizant — CHANGES_REQUESTED** [review]

File: src/main/common/middlewares/agreementservice/chooseagreement.ts (hunk 1 above)
Comment (CesarPerez) [reply src/main/common/middlewares/agreementservice/chooseagreement.ts:16]:
```suggestion
        let BaseURL = "/agreements/RM6263/lots";
```

File: src/main/features/dashboard/controller/dashboard.ts (hunk 2 above)
Comment (CesarPerez) [reply src/main/features/dashboard/controller/dashboard.ts:12]:
```suggestion
  res.cookie('agreement_id', 'RM6263', { maxAge: 900000, httpOnly: true });
```

**Review by prathap-cognizant — CHANGES_REQUESTED** [review]
Can you add unit test cases for your code changes.

**Review by prathap-cognizant — APPROVED** [review]

**Review by viswa-cognizant — APPROVED** [review]
Looks Good

**Pull request merged**